hadoop-hdds/framework/src/test/java/org/apache/hadoop/ozone/audit/TestOzoneAuditLogger.java

```diff
@@ -183,11 +183,11 @@ public void verifyDefaultLogLevelForAuthFailure() throws IOException {
   @Test
   public void messageIncludesAllParts() {
     String message = WRITE_FAIL_MSG.getFormattedMessage();
-    assertTrue(message.contains(USER), message);
-    assertTrue(message.contains(IP_ADDRESS), message);
-    assertTrue(message.contains(DummyAction.CREATE_VOLUME.name()), message);
-    assertTrue(message.contains(PARAMS.toString()), message);
-    assertTrue(message.contains(FAILURE.getStatus()), message);
+    assertThat(message).withFailMessage(message).contains(USER);
+    assertThat(message).withFailMessage(message).contains(IP_ADDRESS);
+    assertThat(message).withFailMessage(message).contains(DummyAction.CREATE_VOLUME.name());
+    assertThat(message).withFailMessage(message).contains(PARAMS.toString());
+    assertThat(message).withFailMessage(message).contains(FAILURE.getStatus());
```

Comment:
Do we need to omit all waitFailMessage or allow the user to partially customize useful messages?   
Because there is still some user-defined information in the current PR

Reply:
I'd remove two kinds:

 * where the string itself is passed (like here)
 * message similar to "expected: ..., got: ..."

We can keep the rest.